test/util/offscreen_texture.test.cpp

```diff
@@ -167,7 +167,11 @@ void main() {
     test::checkImage("test/fixtures/offscreen_texture/render-to-fbo", image, 0, 0);
 
     // Now, composite the Framebuffer texture we've rendered to onto the main FBO.
+#if MLN_LEGACY_RENDERER
     gl::bindTexture(context, 0, {texture.getTexture().getResource(), gfx::TextureFilterType::Linear});
+#else
+    // TODO: Implement based on gfx::Texture2D
```

Comment:
Could you disable this test? Letting it pass without an implementation under test can end up being forgotten about.

Reply:
That makes the test fail, so it shouldn't fall through the cracks.